zulipterminal/ui_tools/views.py

```diff
@@ -1030,7 +1031,9 @@ def __init__(self, controller: Any, title: str, *, zt_version: str,
                 ('Theme', theme_name),
                 ('Autohide', 'enabled' if autohide_enabled else 'disabled'),
                 ('Maximum footlinks', str(maximum_footlinks)),
-                ('Color depth', str(color_depth))])
+                ('Color depth', str(color_depth)),
+                ('Notifications',
+                 'enabled' if notify_enabled else 'disabled')])
```

Comment:
You added some good trailing commas; it'd be good if we could have one here too.

Reply:
I have added the trailing comma there as requested, but note that it won't make the diff smaller on adding extra fields(?), since linter wanted me to add extra whitespace after the comma. :)